todo_app/app.py

```diff
@@ -8,162 +8,215 @@
+            {'idBoard': todoBoard}, {"name": 1})
+        doing = trello_collection.find(
+            {'idBoard': doingBoard}, {"name": 1})
+        done = trello_collection.find(
+            {'idBoard': doneBoard}, {"name": 1})
+
+        my_items = []
+        doing_objects = []
+        done_objects = []
+
+        for todo in trello_collection.find({'idBoard': todoBoard}):
+            my_items.append(ToDo.from_mongo_db_entry(todo))
+        for doing in trello_collection.find({'idBoard': doingBoard}):
+            doing_objects.append(ToDo.from_mongo_db_entry(doing))
+        for done in trello_collection.find({'idBoard': doneBoard}):
+            done_objects.append(ToDo.from_mongo_db_entry(done))
+        view_model = ViewModel(my_items, doing_objects, done_objects)
+        view_model.show_all_done_items()
+        user = current_user.role
+
+        return render_template("index.html", my_items=my_items, doing_objects=doing_objects, done_objects=done_objects, view_model=view_model, user=user)
+
+    @app.route('/create', methods=['POST'])
+    @login_required
+    def create():
+        print(current_user.id)
+        title = request.form.get('title')
+        create_items(title)
+
+        return redirect(url_for('index'))
+
+    @app.route('/complete_item', methods=['PUT'])
+    def complete_item(id):
+
+        print(id)
+
+        update_item(id)
+        return redirect(url_for('index'))
+
+    @app.route('/update', methods=['POST'])
+    def update():
+        app.logger.info('Processing default request')
+        id = request.form.get('id')
+        print(id)
+
+        update_item(id)
+
+        return complete_item(id)
+
+    @app.route('/return_item', methods=['PUT'])
+    def return_item(n):
+        app.logger.info('Processing default request')
+        print(n)
+
+        return_todo(n)
+        return redirect(url_for('index'))
+
+    @app.route('/update_back', methods=['POST'])
+    def update_back():
+
+        n = request.form.get('n')
+        print(n)
+        return return_item(n)
+
+    @app.route('/callback')
+    def login_callback():
+        client = WebApplicationClient(webClient)
+        code = request.args.get("code")
+        client_secret = clientSecret
+        (url, headers, body) = client.prepare_token_request(
+            'https://github.com/login/oauth/access_token', code=code, client_secret=client_secret)
+        get_token_request = requests.post(url, headers=headers, data=body)
+        parse_body = client.parse_request_body_response(get_token_request.text)
+        (url, headers, body) = client.add_token("https://api.github.com/user")
+
+        request_text = requests.get(url, headers=headers, data=body).text
+
+        json_data = json.loads(request_text)
+        id = json_data["id"]
+        if id == githubId:
+            role = "writer"
+        else:
+            role = "reader"
+        user = User(id, role)
+        login_user(user, remember=False, duration=None,
```

Comment:
You're just using the default values for remember, duration, force and fresh. So you could just write `login_user(user)`

Reply:
Updated, thank you